### DISCUSSION TITLE
Preparing a DevOps policy for branch naming

### DISCUSSION THREAD

**Discussion by NuwanJ** [cepdnaclk/smart-inventory-management-system#20] [General]
Shall we prepare a policy on how to name the branches created for each task assigned?

@DDilshani Can you suggest a good one, based on your experience?

**Comment by DDilshani** [comment]
To improve the clarity and formalize the work, follow these naming standards.
Create a branch for each of these with the naming conventions.

- product backlog item/ issue  (PBI) (ex:  issue_##)
- bug fixes (ex: bugfix/bugName)
- unit tests (ex: unitTest/taskName)

Creating pull requests with this naming conventions.
- [taskNumber]-taskName
- [bugfix]-bugName
- [unitTest]-sprint1-tests
